bin/steps/python

```diff
@@ -179,7 +179,12 @@ if [[ -f "$BUILD_DIR/Pipfile" ]]; then
   # The buildpack is pinned to old pipenv, which requires older pip.
   # Pip 9.0.2 doesn't support installing itself from a wheel, so we have to use split
   # versions here (ie: installer pip version different from target pip version).
-  PIP_VERSION='9.0.2'
+  if [ -z "$PIP_VERSION_OVERRIDE" ]
```

Comment:
Unfortunately this won't work, since `$PIP_VERSION_OVERRIDE` will never be set here, since app-provided env vars are passed in via `$ENV_DIR`.